dashboarddoc.md

```diff
@@ -10,14 +10,44 @@ User Dashboard Interaction
 This document illustrates how a Solar Forecast Arbiter user may use the
 Dashboard to complete the following actions:
 
--   [Create a new Site](#create-a-site).
+-   [Create New Site](#create-new-site).
 
--   [Create a new Observation or Forecast](#create-an-observation-or-forecast).
+-   [Create New Observation or Forecast](#create-new-observation-or-forecast).
 
 -   [Upload Data to an Observation or Forecast](#upload-data).
 
+### User Guidance Considerations
+To assist in the user in providing the correct information and
+navigating the dashboard's interface, we've adopted the following
+features not otherwise illustrated in this guide.
 
-Create a Site
+-  To reduce clutter, the dashboard will utilize *progressive
+   disclosure* to hide unnecessary fields from view e.g. the
+   Site form will not display the *PV Modeling Paremeters* fields
+   unless a *Site Type* of *Power Plant* is selected.
+
+-  Descriptive tooltips will be displayed when a user selects a field.
+   <img class="my-3" src="/images/TooltipDetail.png" class="figure"/>
+   <figcaption class="figure pb-2">The latitude field is selected and displays
+   a tooltip describing the data a user should enter.</figcaption>
+
+-  Basic validation will be performed on the entered data to provide
+   immediate feedback to the user. There are two possible sources of
+   error in dashboard interaction:
```

Comment:
what about insufficient input? should that be covered here? a reviewer asked if we'd assume default values and I think the safe answer is "no, they must be entered".